packages/webpack-extraction-plugin/src/GriffelCSSExtractionPlugin.ts

```diff
@@ -177,6 +176,22 @@ export class GriffelCSSExtractionPlugin {
     }
 
     compiler.hooks.compilation.tap(PLUGIN_NAME, compilation => {
+      // WHAT?
+      //   Adds a callback to the loader context
+      // WHY?
+      //   Allows us to register the CSS extracted from Griffel calls to then process in a CSS module
+      const cssByModuleMap: Record<string, string> = {};
+      NormalModule.getCompilationHooks(compilation).loader.tap(PLUGIN_NAME, (loaderContext, module) => {
+        (loaderContext as SupplementedLoaderCotext)[GriffelCssLoaderContextKey] = {
+          registerExtractedCss(css: string) {
+            cssByModuleMap[module.resource] = css;
+          },
+          getExtractedCss() {
+            return cssByModuleMap[module.resource] ?? '';
+          },
+        };
+      });
```

Comment:
```suggestion
      const cssByModuleMap = new Map<string, string>();

      NormalModule.getCompilationHooks(compilation).loader.tap(PLUGIN_NAME, (loaderContext, module) => {
        const resourcePath = module.resource;

        (loaderContext as SupplementedLoaderCotext)[GriffelCssLoaderContextKey] = {
          registerExtractedCss(css: string) {
            cssByModuleMap.set(resourcePath, css);
          },
          getExtractedCss() {
            const css = cssByModuleMap.get(resourcePath) ?? '';
            cssByModuleMap.delete(resourcePath);

            return css;
          },
        };
      });
```

Let's use there `Map` and dispose it once CSS was used.